### PR TITLE
Use BamReadcount version 0.7 in VariantReporting

### PR DESCRIPTION
**Pull request by davidlmorton**

### CODE REVIEW

File: lib/perl/Genome/VariantReporting/Command/ShowDocs.t.expected-output/experts/bam-readcount [hunk 1]
```diff
@@ -7,16 +7,15 @@ Add bam readcount information to a vcf
...
+    max depth to avoid excessive memory. This is the -d parameter.
   [1mminimum_base_quality[0m   Integer
-    don't include reads where the base quality is less than this. This is the -b parameter. This is
-    only available in versions 0.3 and later.
+    don't include reads where the base quality is less than this. This is the -b parameter.
   [1mminimum_mapping_quality[0m   Integer
     filter reads with mapping quality less than this. This is the -q parameter.
   [1mper_library[0m   Boolean
-    report results per library. This is the -p parameter in version 0.5.
+    report results per library. This is the -p parameter
```
Comment (tmooney) [inline lib/perl/Genome/VariantReporting/Command/ShowDocs.t.expected-output/experts/bam-readcount]: The `.` got lost from the end of the sentence here.

File: lib/perl/Genome/VariantReporting/Suite/BamReadcount/Adaptor.pm [hunk 2]
```diff
@@ -24,24 +26,43 @@ class Genome::VariantReporting::Suite::BamReadcount::Adaptor {
...
         },
         max_count => {
             is  => 'Integer',
-            doc => "max depth to avoid excessive memory. This is the -d parameter in version 0.5.",
+            doc => "max depth to avoid excessive memory. This is the -d parameter.",
         },
         per_library => {
             is  => 'Boolean',
-            doc => "report results per library. This is the -p parameter in version 0.5.",
+            doc => "report results per library. This is the -p parameter",
```
Comment (tmooney) [inline lib/perl/Genome/VariantReporting/Suite/BamReadcount/Adaptor.pm]: (Well, the `.` really disappeared from here.)

File: lib/perl/Genome/VariantReporting/Suite/BamReadcount/Adaptor.pm [hunk 3]
```diff
@@ -24,24 +26,43 @@ class Genome::VariantReporting::Suite::BamReadcount::Adaptor {
...
+    my ($self, $params) = validate_pos(@_, 1, 1);
+
+    my @errors = $self->SUPER::__planned_output_errors__($params);
+
+    if (version->parse("v".$params->{version}) < version->parse("v$MIN_VERSION")) {
+        push @errors, UR::Object::Tag->create(
+            type => 'error',
+            properties => ['version'],
+            desc => sprintf("The BamReadcount expert requires version (%s) ".
+                    "or higher, you supplied version (%s).",
```
Comment (tmooney) [inline lib/perl/Genome/VariantReporting/Suite/BamReadcount/Adaptor.pm]: This is a [comma splice](http://en.wikipedia.org/wiki/Comma_splice).  (I don't know how prescriptive we're being about English style.  I tend to be pickier about that than Perl style.)

**Comment by tmooney** [comment]
+1 if the `.` is relocated!

**Comment by nnutter** [comment]
I second Tom's +1 with same caveat.

**Pull request merged**